Makefile

```diff
@@ -2,6 +2,7 @@ VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
 COMMIT := $(shell git log -1 --format='%H')
 GOBIN ?= $(GOPATH)/bin
 GOSUM := $(shell which gosum)
+DOCKER := $(shell which docker)
```

Comment:
To install Docker!

Reply:
Yeah. The `tendermintdev/sdk-proto-gen` image is used, but I coudn't find its Dockerfile. When I inspect the image using [dive](https://github.com/wagoodman/dive), I could find that the image contains the following layers:
![image](https://user-images.githubusercontent.com/5462944/118927253-7a11c400-b97c-11eb-9e21-dad4c3ad5f90.png)
But, it's not clear yet. Let's use this docker strategy for now, and consider alternative ways after making Stargate work.